spec/02-integration/05-proxy/09-balancer_spec.lua

```diff
@@ -1076,8 +1076,8 @@ dao_helpers.for_each_dao(function(kong_config)
         local requests = upstream2.slots * 2 -- go round the balancer twice
 
         -- setup target servers
-        local server1 = http_server(timeout, localhost, PORT + 2, { requests }, true)
-        local server2 = http_server(timeout, localhost, PORT + 3, { requests }, true)
```

Comment:
@Tieske @hishamhm Any objection with removing the `test_log` argument from there?

Reply:
no objection on my part! (@Tieske added that feature for debugging, and it proved very useful to me, so _having_ the argument in that function is a good thing, but it can be kept off by default at all times, leaving it on was a slip)

Reply:
@hisham exactly, there is the constant at the module top that can be used to override the default and turn it on for all tests. See https://github.com/Kong/kong/blob/504503a43decd45a75ace425e5ab77051e1b16b2/spec/02-integration/05-proxy/09-balancer_spec.lua#L50